group_vars/all

```diff
@@ -1,6 +1,7 @@
 ---
 cluster_name: swarm-bharat
 cluster_user: fedora
+cluster_owner: bharat
```

Comment:
Nit: In general, role variables should be prefixed with rolename to prevent clashes in ansible's global namespace.